packages/insomnia-app/app/ui/components/modals/cookie-modify-modal.tsx

```diff
@@ -214,70 +210,83 @@ class CookieModifyModal extends PureComponent<Props, State> {
-                  <button>Raw</button>
-                </Tab>
-              </TabList>
-              <TabPanel>
-                <div className="pad">
-                  <div className="form-row">
-                    {this._renderInputField('key')}
-                    {this._renderInputField('value')}
-                  </div>
-                  <div className="form-row">
-                    {this._renderInputField('domain')}
-                    {this._renderInputField('path')}
-                  </div>
-                  {this._renderInputField(
-                    'expires',
-                    isNaN(new Date(cookie.expires || 0).getTime()) ? 'Invalid Date' : null,
-                  )}
+            <Fragment>
+              <div className="pad">
+                <div className="form-row">
+                  {this._renderInputField('key')}
+                  {this._renderInputField('value')}
                 </div>
-                <div className="pad no-pad-top cookie-modify__checkboxes row-around txt-lg">
-                  {checkFields.map((field, i) => {
-                    const checked = !!cookie[field];
-                    return (
-                      <label key={i}>
-                        {CookieModifyModal._capitalize(field)}
-                        <input
-                          className="space-left"
-                          type="checkbox"
-                          name={field}
-                          defaultChecked={checked || false}
-                          onChange={e => this._handleChange(field, e)}
-                        />
-                      </label>
-                    );
-                  })}
+                <div className="form-row">
+                  {this._renderInputField('domain')}
+                  {this._renderInputField('path')}
                 </div>
-              </TabPanel>
-              <TabPanel className="react-tabs__tab-panel pad">
+                {this._renderInputField('expires', this.validateExpires())}
+              </div>
+              <div className="pad no-pad-top cookie-modify__checkboxes row-around txt-lg">
+                {checkFields.map(field => {
+                  const checked = !!cookie[field];
+                  return (
+                    <label key={field}>
+                      {CookieModifyModal._capitalize(field)}
+                      <input
+                        className="space-left"
+                        type="checkbox"
+                        name={field}
+                        defaultChecked={checked || false}
+                        onChange={e => this._handleChange(field, e)}
+                      />
+                    </label>
+                  );
+                })}
+              </div>
+            
+              <div className="pad">
                 <div className="form-control form-control--outlined">
                   <label>
-                    Raw Cookie String
+                  Raw Cookie String
```

Comment:
I've hit this snag too, is there a lint rule we could use?

```suggestion
                    Raw Cookie String
```

Reply:
fixed in https://github.com/Kong/insomnia/pull/3860 and 5975f816abe3d29f1236183a21b39a73c5114efb